src/util/login.js

```diff
@@ -0,0 +1,51 @@
+import * as firebase from 'firebase';
+
+/** Opens login prompt for user and redirects them to the home page if successful. */
+function promptForLogin() {
+  var provider = new firebase.auth.GoogleAuthProvider();
+  firebase.auth().signInWithPopup(provider).then(result => {
+    var user = result.user;
```

Comment:
Change all these vars to either const or let (preferably const but if you reassign it will need to be let).